src/coreclr/jit/codegencommon.cpp

```diff
@@ -1503,10 +1503,11 @@ void CodeGen::genExitCode(BasicBlock* block)
 void CodeGen::genJumpToThrowHlpBlk(emitJumpKind jumpKind, SpecialCodeKind codeKind, BasicBlock* failBlk)
 {
     bool useThrowHlpBlk = compiler->fgUseThrowHelperBlocks();
-#if defined(UNIX_X86_ABI) && defined(FEATURE_EH_FUNCLETS)
+#if defined(UNIX_X86_ABI)
+    // TODO: Is this really UNIX_X86_ABI specific? Should we guard with compiler->UsesFunclets() instead?
```

Comment:
This should be double-checked along with https://github.com/dotnet/runtime/blob/ce6cf4b2e2a8c7d80ed7b31487eaab574d9007fa/src/coreclr/jit/morph.cpp#L14299-L14304

Is any of that relevant for win-x86 with funclets?